src/threading/thread.cpp

```diff
@@ -311,7 +311,7 @@ bool Thread::bindToProcessor(unsigned int proc_number)
 
 bool Thread::setPriority(int prio)
 {
-#if USE_WIN_THREADS
+#ifdef _WIN32
```

Comment:
see whether the `#if USE_WIN_THREADS` conditions need to be renamed or the code bock removed in this and the other function